app/services/email_digest_article_collector.rb

```diff
@@ -1,39 +1,47 @@
-    return [] unless should_receive_email?
-
-    articles = if user_has_followings?
-                 experience_level_rating = (@user.experience_level || 5)
-                 experience_level_rating_min = experience_level_rating - 3.6
-                 experience_level_rating_max = experience_level_rating + 3.6
-
-                 @user.followed_articles
-                   .select(:title, :description, :path)
-                   .published
-                   .where("published_at > ?", cutoff_date)
-                   .where(email_digest_eligible: true)
-                   .where.not(user_id: @user.id)
-                   .where("score > ?", 12)
-                   .where("experience_level_rating > ? AND experience_level_rating < ?",
-                          experience_level_rating_min, experience_level_rating_max)
-                   .order(score: :desc)
-                   .limit(6)
-               else
-                 Article.select(:title, :description, :path)
-                   .published
-                   .where("published_at > ?", cutoff_date)
-                   .where(featured: true, email_digest_eligible: true)
-                   .where.not(user_id: @user.id)
-                   .where("score > ?", 25)
-                   .order(score: :desc)
-                   .limit(6)
-               end
-
-    articles.length < 3 ? [] : articles
+    # rubocop:disable Metrics/BlockLength
+    instrument ARTICLES_TO_SEND, tags: %W[user_id:#{@user.id}] do
+      return [] unless should_receive_email?
+
+      articles = if user_has_followings?
+                   experience_level_rating = (@user.experience_level || 5)
+                   experience_level_rating_min = experience_level_rating - 3.6
+                   experience_level_rating_max = experience_level_rating + 3.6
+
+                   @user.followed_articles
+                     .select(:title, :description, :path)
+                     .published
+                     .where("published_at > ?", cutoff_date)
+                     .where(email_digest_eligible: true)
+                     .where.not(user_id: @user.id)
+                     .where("score > ?", 12)
+                     .where("experience_level_rating > ? AND experience_level_rating < ?",
+                            experience_level_rating_min, experience_level_rating_max)
+                     .order(score: :desc)
+                     .limit(6)
+                 else
+                   Article.select(:title, :description, :path)
+                     .published
+                     .where("published_at > ?", cutoff_date)
+                     .where(featured: true, email_digest_eligible: true)
+                     .where.not(user_id: @user.id)
+                     .where("score > ?", 25)
+                     .order(score: :desc)
+                     .limit(6)
+                 end
+
+      articles.length < 3 ? [] : articles
+    end
+    # rubocop:enable Metrics/BlockLength
   end
```

Comment:
This is what the diff looks like when you exclude whitespace-only changes. I'm literally just wrapping the existing code in a block. 🙂

<img width="516" alt="Diff without whitespace-only changes" src="https://user-images.githubusercontent.com/108205/112738239-7858f080-8f37-11eb-85e9-a399906fdf46.png">